main.go

```diff
@@ -127,19 +125,23 @@ func enableOnCI() {
 }
 
 func setPalette() {
-	v := os.Getenv(paletteEnv)
-	if v == "" {
-		return
-	}
-	vals := strings.Split(v, ",")
-	if len(vals) != 2 {
-		return
-	}
-	if c, ok := colors[vals[0]]; ok {
-		fail = color.New(c)
-	}
-	if c, ok := colors[vals[1]]; ok {
-		success = color.New(c)
+	envArray := [3]string{"GOTEST_SKIP_COLOR", "GOTEST_FAIL_COLOR", "GOTEST_PASS_COLOR"}
```

Comment:
These variable names should be constants

Reply:
Constants added